contracts/get_selector.ts

```diff
@@ -0,0 +1,7 @@
+import { getSelectorFromName } from 'starknet/dist/utils/hash'
```

Comment:
This looks like and ad-hoc one-off script, one that's not really useful anymore.

Please translate it to a unit test, so we'll have it there as an example how to compute fn selector.